components/brave_vpn/resources/panel/components/select-region-list/index.tsx

```diff
@@ -31,27 +82,111 @@ function RegionContent(props: RegionContentProps) {
     if (props.selected) ref.current?.scrollIntoView()
   }, [])
 
+  const countryInfo = {
+    $1: props.region.cities.length,
+    $2: props.region.serverCount
+  }
+
   return (
     <S.RegionContainer
       selected={props.selected}
+      fillBackground={!props.open}
       ref={ref}
     >
-      <S.RegionCountry>
+      <S.RegionCountry
+        onClick={() => {
+          // Pass '' to toggle currently opened country.
+          props.clickHandler(props.open ? '' : props.region.name)
+        }}
+      >
         <Flag countryCode={props.region.countryIsoCode} />
-        <S.RegionCountryLabel>
-          {props.region.namePretty}
-        </S.RegionCountryLabel>
+        <S.CountryInfo>
+          <S.RegionCountryLabel selected={!props.open && props.selected}>
+            {props.region.namePretty}
+          </S.RegionCountryLabel>
+          <S.CountryServerInfo selected={!props.open && props.selected}>
+            {getLocale('braveVpnServerSelectionCountryInfo').replace(
```

Comment:
@simonhong this seems pretty useful - maybe worth adding the function to `components/common/locale.ts` (seems much more usable than the existing tags logic)

```ts
export function getLocaleWithReplacements(key: string, replacements: { [key: `${number}` ]: string }) {
    return getLocale(key).replace(/\$\d+/g, m => replacements[m])
}